data/zilliqa/assets/zil19cyp6kj7zrz3c2q50ts2nly6hrhrr7vr89g9gf/logo.svg

```diff
@@ -0,0 +1,292 @@
+hmu+x0Q5B0EEDKysSfGbTzePyTuNtYr54GQBUiScybYnQf8GZ/RF3iDnpWxIirBH3WyIlgiJDNaj
+sHy4Hs4ae/vY/AYVKliSG3z08zfbgCCIBabmAjwiy5tS1qYPznZq5MrVPeHNa/5z4cRB5IieLNfJ
+oKvtnLn1ePn+3Wa+QYUGlnR+iv26DSZV48FSE8JA7gJ9TvVBGZUuqfZUj45ElyZkwJdKG4TZl0iN
+oVVzxNpZrQnjb8tqfc/XbgtYFNMvFyl59ZM+tgiACCdgIFwg+9SWgfigeNVFwe9BvCCD/FgDcd6g
+SKVLoQi2o4b8qXtVFOvt5dv5kmBlSMALiz/19Wd0XnrAwGk7qVn4R6WfgmzX22SJOVcO4lRSBTtx
+4iDG2kOvyhl0MP6aEm7+ZXbDTPdChePDnXf8f5R+vgEG6o7FZ4oLAU5wrly5LEP6Ho6NN3j8IFLp
+5bIz6MT48C36NXqqfBMq/Mnmp2o3fWBgV5aB+wALgU4wqlwRlfDuV6UNkjiIInklWG4UOhof/39B
+y8J58jhU+OYX2O/7IOuC+9oOVdeK7gMzQrxAhUJDQmyoIcwWnIS6hWJ7/P/Uerz8m+Od2n0/UvpL
+zM8eBY6HImO7i3hoO2vMFPXPP8IRhMLgiu1rQmUwHGgZ1StTp7UjUnrHTNPM/0Wbv0/5p5MBAxG5
+Ox6KdbDfzrxRMgWnRAg++EE+YjhHw8Y/JvtmXwnXghVWOT0jRUqfqtI2lKNAx7+F7K0HD6UPGqTF
+yoDcKbExOkE+bvyU/5fgHJwZjcR6R0oNKhz2rPOnu2BF8FCUbtwHcp2fh/tcKLKlZBgAt8NHdYFP
+CCCldp+3BgIIsx6FJkFKV16S+b0uoAyOYMJV+OCOAd1Xu9v2dbvz0/aP/4tDnpHnFjeEEPIO1lcF
+J6YMz4szP7pAVLCIBRITFlycIb1EdhWKszbOceLzc85P/CdcDUTJGTNpEiEoOauM3Po5kyR79ZI+
+Ez4HzTHh4AOKwsAJTc695fhf4uIExq8q/F0EyRkcIExCuMh5OZSdofVpnfnXH33fdEGPDw4kra9b
+hgM3IOgLYR9+2Wgc+XUr4aGh5JzojULCQY/1L9P8zE6ImLAUdsJKUFw8A4s/MsSbuZJmzqwftCJK
+bljTS7Xel0Hqg7nNJa/wSKovsHj18SlZhMmlaKAVpCUTHxv5GzHCl+qDRVfJghfyAot/Y0rajMDk
+ea+eaSH1d7K+1wfj7oRN0ctyWPyROXkRJt/yKnJOUfLfyPqMD8wJ9EIhgcU3HhuWbDB5IcERXILy
+1W/3UaFIcYKMCc3A4v2zMhUmHxszwUEBVn9L690HE6liZB8s3jMs1WFyB8knhm39X8LJfEJArmcg
+AAAAAElFTkSuQmCC">
+</image>
+<linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="-11.6125" y1="5.4248" x2="276.6807" y2="258.5368">
+	<stop  offset="0.2158" style="stop-color:#242A5E"/>
+	<stop  offset="0.2439" style="stop-color:#283366"/>
+	<stop  offset="0.2891" style="stop-color:#324A7C"/>
+	<stop  offset="0.3456" style="stop-color:#4370A0"/>
+	<stop  offset="0.4107" style="stop-color:#5AA5D1"/>
+	<stop  offset="0.4132" style="stop-color:#5BA7D3"/>
+	<stop  offset="0.4937" style="stop-color:#71AFD6"/>
+	<stop  offset="0.6247" style="stop-color:#508DBB"/>
+	<stop  offset="0.7135" style="stop-color:#242A5E"/>
+</linearGradient>
+<path class="st0" d="M213.6,42.4C190.7,19.6,160.3,7,128,7S65.3,19.6,42.4,42.4C23.7,61.2,11.8,85.1,8.2,111
+	C7.4,116.6,7,122.3,7,128c0,32.3,12.6,62.7,35.4,85.6C65.3,236.4,95.7,249,128,249s62.7-12.6,85.6-35.4
+	c18.8-18.8,30.6-42.7,34.3-68.5c0.8-5.6,1.2-11.3,1.2-17C249,95.7,236.4,65.3,213.6,42.4z M237.9,145
+	c-8.2,53.2-54.4,94.1-109.9,94.1c-49.4,0-91.4-32.4-105.8-77.1h23.2c13.4,32.5,45.3,55.3,82.6,55.3c43.5,0,79.8-31.1,87.7-72.3
+	h-35.1c-2.6,8.2-7.2,15.8-13.5,22.1c-10.4,10.4-24.3,16.2-39.1,16.2s-28.7-5.8-39.1-16.2c-6.3-6.3-10.9-13.8-13.5-22.1
+	c-1.8-5.4-2.7-11.2-2.7-17H16.8c0-5.8,0.4-11.5,1.3-17C26.4,57.7,72.5,16.8,128,16.8c49.4,0,91.4,32.4,105.8,77.1h-23.2
+	c-13.4-32.5-45.3-55.3-82.6-55.3c-43.5,0-79.8,31.1-87.7,72.3h35.1c2.6-8.2,7.2-15.8,13.5-22.1c10.4-10.4,24.3-16.2,39.1-16.2
+	s28.7,5.8,39.1,16.2c6.3,6.3,10.9,13.8,13.5,22.1c1.8,5.4,2.7,11.2,2.7,17h55.9C239.2,133.8,238.7,139.5,237.9,145z"/>
+</svg>

```

Comment:
Rather than encoding your png in a svg, provide a normal png.

Reply:
It was a shine overlay. can't be done with pure svg. Replaced with png